servicetest/testframework/lib.py

```diff
@@ -262,7 +268,7 @@ def terminate(self):
 
     def __create_container(self, config):
         self.__container_id, success = self.__agent.Create(config)
-        return success
+        return (self.__container_id, success)
```

Comment:
What doesn't work is this:
        return self.__agent.Create(config)
But I can remove the parenthesis and return more than one value.